### ISSUE TITLE
Implement mkproject command/script

### ISSUE THREAD

**Issue by khorn** [davidmarble/virtualenvwrapper-win#45]
`virtualenvwrapper` implements a `mkproject` command which creates a project dir, creates an environment variable, and connects them with it's equivalent of `setproject`.

This is a nice shortcut, and `virtualenvwrapper-win` should have it.

**Comment by utek** [comment]
Does it also creates virtualenv? It would be nice if you could write what current command mkproject is equivalent for. Thx.

**Comment by amullins83** [comment]
@utek Yes, the virtualenvwrapper command creates a project directory and a virtualenv with the same name, and sets the new project folder as the active project for that new virtualenv. It's made fairly clear in the virtualenvwrapper tutorial here: http://virtualenvwrapper.readthedocs.io/en/latest/command_ref.html#project-directory-management
